### PR TITLE
Redo elyses-enchantments/lists.py as card-games

### PR DESCRIPTION
**Pull request by IsaacG**
Rewrite the lists concepts exercise

### CODE REVIEW

**Comment by BethanyG** [comment]
Please see #2404, (_which should be merged first or combined with this rename_)  for the [associated `introduction.md`](exercises/concept/elyses-enchantments/.docs/introduction.md).  The `introduction.md` file included here is outdated.

**Comment by BethanyG** [comment]
@cmccandless  - tagging you because I extensively helped with the story on this, so I think it needs a different set of eyes.  😄

File: exercises/concept/card-games/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## Instructions
+
+Elyse is really looking forward to playing some poker (and other card games) during her upcoming trip to Vegas. Being a big fan of "self-tracking" she wants to put together some small functions that will help her with tracking tasks and has asked for your help thinking them through.
+
+## 1. Tracking Poker Rounds
+
+Elyse is especially fond of poker, and wants to track how many rounds she plays - and _which rounds_ those are. Every round has its own number, and every table shows the round number currently being played. Elyse chooses a table and sits down to play her first round. She plans on playing three rounds.
+
+Implement a function `to_rounds` that takes the current round number and returns a single list with that round and the _next two_ that are coming up:
```
Comment (cmccandless) [inline exercises/concept/card-games/.docs/instructions.md:11]: I had to read this a couple times to understand what `to_rounds` was doing. What about `get_rounds`?

Comment (BethanyG) [reply exercises/concept/card-games/.docs/instructions.md:11]: Agree.  `get_rounds` reads better.  @IsaacG ?

Comment (IsaacG) [reply exercises/concept/card-games/.docs/instructions.md:11]: No preference. Updating...

**Comment by BethanyG** [comment]
@IsaacG  -- if @cmccandless has no additional comments, I think this is good to go!  🎉   **_thank you_** for your work on it.  

I am, however going to wait to merge-- because we've renamed the exercise from what it was originally, we'll probably need @iHiD  help with syncing.  And since there are at least three more exercises that need rewrite/rename, I think we'll want to do them in a batch to avoid problems.

**Review by cmccandless — APPROVED** [review]

**Comment by cmccandless** [comment]
Approved

**Comment by iHiD** [comment]
> I am, however going to wait to merge-- because we've renamed the exercise from what it was originally, we'll probably need @iHiD help with syncing. And since there are at least three more exercises that need rewrite/rename, I think we'll want to do them in a batch to avoid problems.

If these are renamed concept exercises, it should be fine to merge. Things should automatically sync up. Thanks for checking though, and if it doesn't tell us so we can fix the bug :)

**Comment by BethanyG** [comment]
Urg.  @cmccandless  - I think I need your `git-foo` here.  Tried to resolve the merge conflicts manually at the CL, but (_maybe because I have a fork??_) I was unsuccessful.

**Comment by cmccandless** [comment]
I'm going to try rebasing onto our main branch

**Comment by cmccandless** [comment]
I'll make a bookmark branch just in case to preserve the work here as it is now

**Comment by cmccandless** [comment]
CI looks good. Can @BethanyG and/or @IsaacG please inspect the rebased branch to ensure work has not been lost?

**Comment by BethanyG** [comment]
I *think* we're good to go.  @IsaacG ?  Spot anything in need of correcting??

**Comment by IsaacG** [comment]
Looks about right to me!

**Pull request merged**